Account information added to login

oguzhan51511 opened the pull request:
After everything is ready, I'm on the slope of transferring the information of the account. Thus, I don't have to send the account number as a parameter in multiple use.

oguzhan51511 committed:
Account information added to login

After everything is ready, I'm on the slope of transferring the information of the account. Thus, I don't have to send the account number as a parameter in multiple use.

pedroslopez commented:
Hi, thanks for contributing.

Is there a reason why you couldn't get the info from `client.info` and use the `authenticated` event for session details?

oguzhan51511 commented:
There is only session information in the authenticated event.
I could not access client.info data. What I need is the number of the account logged in. If there is any place I can reach, it works.

oguzhan51511 committed:
Update Client.js

I guess it's more convenient like this. because I don't need session information.

oguzhan51511 committed:
Merge branch 'master' into patch-1
